build.gradle

```diff
@@ -27,6 +27,10 @@ dependencies {
 android {
     compileSdkVersion 26
 
+    defaultConfig {
+        minSdkVersion 14
```

Comment:
Why have you moved the `minSdkVersion` and not the `compileSdkVersion`? Is there any difference between setting this in **build.gradle** or in **AndroidManifest.xml**?

Reply:
most recent Android Studio complain about having this values in `AndroidManifest.xml` It's a relict form Eclipse, which is not more supported since ~ 2016.09
It should be located in 'build.gradle'

It's related to https://github.com/owncloud/android-library/pull/180 which removes Eclipse (or did it as I created the PR long time ago)

Reply:
I see, can you move also the targetSdkVersion here?